CIPs/0022.md

```diff
@@ -0,0 +1,146 @@
+# CIP [0022]: Upgrade epoch SNARK data for flexibility, slashability and security
```

Comment:
Hi @kobigurk I'd remove this title as the preamble will have all the information needed. Preamble will scraped later with a CIP website that I will integrate later.